sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -337,6 +337,32 @@ object RapidsConf {
     .bytesConf(ByteUnit.BYTE)
     .createWithDefault(0)
 
+  val OFF_HEAP_LIMIT_ENABLED = conf("spark.rapids.memory.host.OffHeapLimit.enabled")
```

Comment:
Nit: convention is we don't capitalize the first word after a `.` in our config names.